### DISCUSSION TITLE
Hitcircles not displaying properly

### DISCUSSION THREAD

**Discussion by slopslug** [ppy/osu#19729] [Q&A] [answered]
### Type

Cosmetic

### Bug description

Hitcircles which have the insides coloured not rendered correctly compared to stable. Could this have something to do with lazer handling hitcircles differently?

Happening/tested on the 2021 Ipad Pro 12.9

[skin](https://www.icloud.com/iclouddrive/0abYpINgmyBQcXrqF73rylgfg)

### Screenshots or videos

# Reference images from stable
![screenshot147](https://user-images.githubusercontent.com/47635446/184431169-65b2a8a8-4647-4ae8-83da-207684edcef5.jpg)
![screenshot148](https://user-images.githubusercontent.com/47635446/184431233-210c291c-97ab-4df5-afbd-48d0cb75e955.jpg)
![](https://i.imgur.com/gpgzZB8.png)
# References from mobile
![IMG_0918(1)](https://user-images.githubusercontent.com/47635446/184431614-ed5d75e2-198c-4947-b106-068ad5db45cc.PNG)
![IMG_0917(1)](https://user-images.githubusercontent.com/47635446/184431652-fbc0fe02-4261-4a47-a10f-e35fc30be59d.PNG)

### Version

2021.810.2 (Supporter testflight group)

### Logs

I cannot copy the folder through iTunes to my computer, it just won't let me. No drag and drop nor copy and paste.

**Comment by peppy** [comment, marked as answer]
You need to attach your skin.

**Reply by slopslug** [reply]
[skin](https://www.icloud.com/iclouddrive/0abYpINgmyBQcXrqF73rylgfg)

**Reply by peppy** [reply]
This is going to be one or more of the following at play:

- Slider tracks looking different, making it hard to see the expected gloss overlay
- Premultiplied alpha being applied incorrectly (if only iOS, which seems like it may be the case)
- osu!stable having incorrect gamma applied (corrected in lazer IIRC)

Will require further investigation.

Relevant files attached.

![hitcircle@2x](https://user-images.githubusercontent.com/191335/184442013-35a3d63c-4f77-4763-9bfe-af075eeb207f.png)
![hitcircleoverlay@2x](https://user-images.githubusercontent.com/191335/184442020-a433b6f8-04d5-4468-b5b8-c03cee4a7b0c.png)

**Reply by peppy** [reply]
Tracking at https://github.com/ppy/osu/issues/19957
